Build failure on Nixos flake

Build failure on Nixos flake module, missing "-lgbm"
error log:
```
error: builder for '/nix/store/z1pmndvigc2kmz2jbjcgvms7paz4dh3q-fht-compositor-8f0c359.drv' failed with exit code 101;
       last 25 log lines:
       > 99 |         adaptive_sync: Option<bool>,
       >    |         ^^^^^^^^^^^^^
       >    |
       >    = note: `OutputConfiguration` has a derived impl for the trait `Debug`, but this is intentionally ignored during de
ad code analysis
       >    = note: `#[warn(dead_code)]` on by default
       >
       > warning: method `active_monitor` is never used
       >    --> src/space/mod.rs:221:12
       >     |
       > 56  | impl Space {
       >     | ---------- method in this implementation
       > ...
       > 221 |     pub fn active_monitor(&self) -> &Monitor {
       >     |            ^^^^^^^^^^^^^^
       >
       > error: linking with `/nix/store/gj9lra51hwhxnhz05jqk5lh03wipamv0-gcc-wrapper-14-20241116/bin/cc` failed: exit status: 
1
       >   |
       >   = note: LC_ALL="C" PATH="/nix/store/464j7y43imzxc94955xgysxxcnadaah0-rustc-1.85.0/lib/rustlib/x86_64-unknown-linux-g
nu/bin:/nix/store/wvcl683adl5vsc2873rdcs255zq3j7yk-cargo-1.85.0/bin:/nix/store/q88m6mq2pbd8nq7dr0p065334nsglqhr-cargo-auditable
-0.6.5/bin:/nix/store/8cczxvv6hpngc9mxal0wx0lzny4vjp4n-pkg-config-wrapper-0.29.2/bin:/nix/store/2b3ahc691h5syn9qp18fkps6hp1n0rd
l-auditable-cargo-1.85.0/bin:/nix/store/fsyj2gv51gcwcs8gx44xix044hb716a5-rustc-wrapper-1.85.0/bin:/nix/store/wvcl683adl5vsc2873
rdcs255zq3j7yk-cargo-1.85.0/bin:/nix/store/qp4a5bxdck7c0w7l2mf47phhfk2nmscs-patchelf-0.15.0/bin:/nix/store/gj9lra51hwhxnhz05jqk
5lh03wipamv0-gcc-wrapper-14-20241116/bin:/nix/store/aw0qxjd1phf16qhlwpdb4x87yymfv9rp-gcc-14-20241116/bin:/nix/store/cgp5cbv6b3d
jw35zkl4s77snh0navr3w-glibc-2.40-66-bin/bin:/nix/store/vz67zaqyfln2jcqwar682kdlrxfx9vq1-coreutils-9.6/bin:/nix/store/kinjsav83q
giiy8mwgypnnc9sn8m8r9y-binutils-wrapper-2.43.1/bin:/nix/store/dh0m67pv8snfcx1ic18jzp0x4cb10zhc-binutils-2.43.1/bin:/nix/store/v
z67zaqyfln2jcqwar682kdlrxfx9vq1-coreutils-9.6/bin:/nix/store/cfd3hysxk7b4m584s539hzm2bc013lsg-findutils-4.10.0/bin:/nix/store/k
drp6y7k352jb9ysjprzvmrlmiblz0jq-diffutils-3.10/bin:/nix/store/l97jblhqk9ga2dl01ysd3wf51mwqcd6s-gnused-4.9/bin:/nix/store/h3xhfh
g0vv3k3hvkcwkmdg99r4zm9mcl-gnugrep-3.11/bin:/nix/store/kz9cfpxf08fdsgxgh6zp1k1y5npsnjpr-gawk-5.3.1/bin:/nix/store/j9gjzsbag34n3
ic3yg4ra0p5yl8wcbhg-gnutar-1.35/bin:/nix/store/g9h265xkvsma74irj4389djs39k96d72-gzip-1.13/bin:/nix/store/p406mc3r448lfrl0qgqm4c
g8qax7jw5v-bzip2-1.0.8-bin/bin:/nix/store/zkman1hvq58ica17s06r9hq62k300c1v-gnumake-4.4.1/bin:/nix/store/ki4if6b0w5bqv8dc5lrjp8x
m7wjy9dlf-bash-5.2p37/bin:/nix/store/gkw3fwmfvn06gz7hh6vwdskh3s1lkvcv-patch-2.7.6/bin:/nix/store/7w6d3dwm5zs6vn37wmsbi0szvx0w98
gy-xz-5.6.4-bin/bin:/nix/store/i4fn3frirjgrfsmwkbr4srfpry18x5dz-file-5.46/bin" VSLANG="1033" "/nix/store/gj9lra51hwhxnhz05jqk5l
h03wipamv0-gcc-wrapper-14-20241116/bin/cc" "-m64" "/build/rustcyFm9WK/symbols.o" "<17 object files omitted>" "-Wl,--as-needed" 
"-Wl,-Bstatic" "/build/17lz53zj5vqxi9335j1h6dqqxvvvaqsc-source/target/x86_64-unknown-linux-gnu/release/deps/{libtracing_subscri
ber-ae7f184e04f00185.rlib,libsharded_slab-77a32c4d0c3a3467.rlib,libmatchers-93495ee9a941d22d.rlib,libregex_automata-cf3fd97db4e
c4dee.rlib,libregex_syntax-e3cf2007c3bd10d4.rlib,libnu_ansi_term-f8dc47b1660334bb.rlib,liboverload-0ff77dbdefa47740.rlib,libthr
ead_local-7f0741544bfea95c.rlib,libtracing_log-41a3a981cb6190b5.rlib,libserde_json-18f404f05df226ab.rlib,libitoa-909812c271ff09
6c.rlib,libryu-157551ed9c691cf8.rlib,libegui_glow-f92392ccdfe79427.rlib,libegui-113d660939d7d59e.rlib,libepaint-1beba6fdc665fc4
e.rlib,libnohash_hasher-bf46769f15d30f34.rlib,libepaint_default_fonts-d68b30aa64438553.rlib,libab_glyph-6bdae780041d0edd.rlib,l
ibowned_ttf_parser-baf10166b82ef97f.rlib,libttf_parser-8e67c370e8091e8d.rlib,libab_glyph_rasterizer-7da13ab2446142b8.rlib,libpa
rking_lot-31a86cb2ae0133ef.rlib,libparking_lot_core-bca4047a14b0e19d.rlib,liblock_api-ba1b64aad2f02d4e.rlib,libecolor-63a2ad124
32a995e.rlib,libemath-93857cef94ae296f.rlib,libclap_complete-650db7bc8aefe7e1.rlib,libtracy_client-7fd7b180947fdda3.rlib,libtra
cy_client_sys-4b9998a0df1dca74.rlib,libowning_ref-c0e8f1e75c5faf32.rlib,libstable_deref_trait-8d87e10d1e967e1f.rlib,libpipewire
-9d3c7477ce87fb1d.rlib,libpipewire_sys-4eca29bd01b7292b.rlib,liblibspa-ebda3c9e19488f6a.rlib,liblibspa_sys-7f4b75ec1c72bca2.rli
b,libcookie_factory-aa4fe101fa728f8f.rlib,libfutures-e1c577bb5c608e90.rlib,libfutures_executor-1961b6956d8c3a61.rlib,libfutures
_util-e3bd3dc0f3236e7e.rlib,libfutures_channel-88fa848e22293b4e.rlib,libfutures_sink-047229788d4dd1b1.rlib,libfutures_task-ceaf
bc938ccbfc45.rlib,libpin_utils-8550a248d588592f.rlib,libnom-b7e25943f085522c.rlib,libconvert_case-ec347c1754743504.rlib,libunic
ode_segmentation-632e5530776f849e.rlib,libglam-629717a83b89e924.rlib,libzbus-b923994b4e9bf6b8.rlib,libblocking-1dc4ffdc076bfef7
.rlib,libpiper-887431849a2a851b.rlib,libatomic_waker-4f66e23fad42b540.rlib,libhex-db0916fc177501c4.rlib,libasync_executor-e299d
ecf59e90d92.rlib,libasync_process-177efa89488f81bf.rlib,libasync_signal-665dbd6974c4519e.rlib,libsignal_hook_registry-d18c42a9a
e43fd50.rlib,libasync_task-c391b8277ea0ee46.rlib,libasync_channel-140cf1a2254824e6.rlib,libordered_stream-8de25f3429136a3e.rlib
,libasync_broadcast-ed4f3cb1d31fe7f3.rlib,libnix-d265d172b6755343.rlib,libasync_io-4077888a6412cd17.rlib,libslab-3ae9b817fd1cd6
a4.rlib,libconcurrent_queue-bd857080d80538aa.rlib,libcrossbeam_utils-b3ddb04d61cbf2ea.rlib,libasync_lock-637da30a3ca3a384.rlib,
libevent_listener_strategy-3e49b0247e3b1da8.rlib,libevent_listener-b2b3441ad107d7da.rlib,libfutures_lite-c876f11e22ba1122.rlib,
libfutures_core-41f35e0d804b155f.rlib,libparking-d879157d0f6dcf48.rlib,libfutures_io-83d70ca19e3d6635.rlib,libzbus_names-4cb6d9
dc85986239.rlib,libzvariant-71374934c9c5998e.rlib,libenumflags2-e040b7a45ce9e710.rlib,libendi-acaf59c643585101.rlib,libzvariant
_utils-bd5856651f7dafab.rlib,libsyn-9276f8b73e23d4cb.rlib,libquote-57958dba3487ff9b.rlib,libproc_macro2-efafd153a199889c.rlib,l
ibunicode_ident-08699ef2b0761ca6.rlib}" "/nix/store/464j7y43imzxc94955xgysxxcnadaah0-rustc-1.85.0/lib/rustlib/x86_64-unknown-li
nux-gnu/lib/{libproc_macro-147648d5254a633d.rlib}" "/build/17lz53zj5vqxi9335j1h6dqqxvvvaqsc-source/target/x86_64-unknown-linux-
gnu/release/deps/{libstatic_assertions-9501f91e25312f3a.rlib,libfht_compositor_config-4bde68ecf1da4cab.rlib,libxdg-362b24e10689
1b9d.rlib,libcsscolorparser-362b662cf27100e0.rlib,libphf-fe8c2eb4bef26cfd.rlib,libphf_shared-2466d04fcabc197d.rlib,libsiphasher
-d419633d5be1e393.rlib,libsscanf-62088ec53655fedd.rlib,liblazy_static-9ed9448dea022d71.rlib,libconst_format-6a9938be594aa8e6.rl
ib,libtoml-2cf88c6268a9e150.rlib,libtoml_edit-ba8b6aa720f48d11.rlib,libserde_spanned-20920b80027387d0.rlib,libwinnow-205af08f50
3f8a28.rlib,libtoml_datetime-934ef0948b77cb36.rlib,libregex-9fb98ad2bcb06979.rlib,libregex_automata-ce2f8c2ce4a90cc8.rlib,libah
o_corasick-e40130f062460819.rlib,libmemchr-1d6462cfae55fe63.rlib,libregex_syntax-2f4197f11cba9cf3.rlib,libfht_animation-8556bef
9267bd83c.rlib,libserde-66f460f4820508a9.rlib,libkeyframe-b5fe842295b1fd2c.rlib,libsmithay_drm_extras-ad0bf5e63a548365.rlib,lib
libdisplay_info-9a71eeb3d9b7c3ad.rlib,libthiserror-e62ab28e71038a39.rlib,liblibdisplay_info_sys-09c6c42f4d2c70be.rlib,libanyhow
-957d738fb098a332.rlib,libsmithay-9333f026d839bdce.rlib,libappendlist-d91b10f0cd46589e.rlib,libtempfile-77d6d17a2fb41989.rlib,l
ibgetrandom-dd1b2f169a0960bb.rlib,libfastrand-bd79ac46f165f756.rlib,libprofiling-d660310c42ada8fb.rlib,libwayland_protocols_mis
c-38723d94b3e08ea0.rlib,librand-a740b922586d94f4.rlib,librand_chacha-80ee2700ecd90360.rlib,libppv_lite86-79382c8237e7fb53.rlib,
librand_core-360602117d3068e3.rlib,libscopeguard-67c0bf9f2abdb719.rlib,liblibseat-8cf0c287a1acaf03.rlib,liberrno-e1547f44309789
db.rlib,liblibseat_sys-dd63f0b9d476e51b.rlib,libinput-71b26fd7c343114a.rlib,libudev-f62aa644ae7bc863.rlib,liblibudev_sys-12afc9
a6d97b2b1c.rlib,libio_lifetimes-1a38292ff383da35.rlib,libinput_sys-6e9bf0170d11bb7d.rlib,libwinit-02c3609783def32a.rlib,libperc
ent_encoding-616e51263ca5ef4f.rlib,libx11rb-475f86bc2e291754.rlib,libas_raw_xcb_connection-b59bfac8baae61a1.rlib,libx11rb_proto
col-569b0a9a8824362c.rlib,libwayland_protocols_plasma-56a393dc08cf23fb.rlib,libahash-f9476d966e3debad.rlib,libgetrandom-ad00327
ab607ce10.rlib,libzerocopy-d3446bf4acab70f4.rlib,libbyteorder-860c154093b8c4a9.rlib,libsmithay_client_toolkit-746053244b7a1ef3.
rlib,libwayland_cursor-0aa5e73bb0b821f7.rlib,libxcursor-f01ced34ab6c1681.rlib,libwayland_protocols_wlr-87e96d83ae88b65b.rlib,li
bwayland_protocols-21bce522dea0b3a9.rlib,libwayland_csd_frame-7eec9eaeab9864f1.rlib,libcalloop_wayland_source-938bf823c620abb2.
rlib,libwayland_client-5edf884a9f779438.rlib,libcalloop-3d124f34396224b5.rlib,libx11_dl-8076dc8e58dee08f.rlib,libxkbcommon_dl-3
f0e755cb730ee62.rlib,libsmol_str-84c7929854728283.rlib,libcursor_icon-29b9eb3875ec606a.rlib,libdpi-1cea900b63c49f5c.rlib,libraw
_window_handle-e0e374da90289393.rlib,libwayland_egl-a5502188d5ca476d.rlib,libthiserror-25a20ed909f7604a.rlib,libaliasable-7595c
f9c79a7de96.rlib,libglow-18060b38db670f75.rlib,libcgmath-1b14d5e215959b81.rlib,libapprox-2be3cd105ecfd99f.rlib,libnum_traits-97
2e60455294d76e.rlib,libxkbcommon-ead4e8bc46445063.rlib,libxkeysym-a55843d81b039fd6.rlib,libmemmap2-7a6a836844ce4c35.rlib,libash
-97fc6d4841d49993.rlib,libgbm-b17e46f94c1e107a.rlib,libwayland_server-28f11a672e567f06.rlib,libwayland_backend-06df9dec627a7816
.rlib,libdowncast_rs-a1b6b78d385abab4.rlib,libscoped_tls-9b4dc2ef4769da1d.rlib,libwayland_sys-a3d51e746cd10ebe.rlib,libmemoffse
t-b7aa3aadd746cf00.rlib,libdlib-78c7bd909bb1c2f3.rlib,liblibloading-45214fbd5c8ce232.rlib,libsmallvec-62f2bac33f4d861a.rlib,lib
gbm_sys-b572d824fdfbff4f.rlib,liblibc-dfe4f598ce5200da.rlib,libindexmap-c9e13dc3de86a0f8.rlib,libequivalent-b9a1db550357db4c.rl
ib,libhashbrown-1ad579a99721c3ee.rlib,libdrm-6dbf81aabd97f81c.rlib,libbytemuck-4007dd5ab5971c23.rlib,libdrm_ffi-af994b06167954f
8.rlib,libdrm_sys-e7f95f60dedb5182.rlib,liblinux_raw_sys-a425ee93d5c38982.rlib,libdrm_fourcc-6c59dd01b0985e7a.rlib,libcalloop-c
d76d21e1ff93293.rlib,libpolling-3e07645f035ee692.rlib,librustix-3abe18c7d16def35.rlib,libbitflags-69ad81d69a0122c4.rlib,liblinu
x_raw_sys-38547da9006c65b8.rlib,libcfg_if-69b6e8f48b5b4283.rlib,libclap-147e3b4cee0cf746.rlib,libclap_builder-5f79a472b882e949.
rlib,libstrsim-dde2d15f47026192.rlib,libanstream-f9a0f8abae8c1608.rlib,libanstyle_query-85e765ae2dd0183c.rlib,libis_terminal_po
lyfill-9421ccafc95565c3.rlib,libcolorchoice-e403685415fd70fa.rlib,libanstyle_parse-fcfd26eab6f0e25f.rlib,libutf8parse-142a91ca8
4878e36.rlib,libclap_lex-c1295b962a73a472.rlib,libanstyle-bcb22ff3d90ea73f.rlib,libtracing-e6d2d589848e3263.rlib,liblog-d679420
287cafebf.rlib,libpin_project_lite-b60b85148bae033b.rlib,libtracing_core-1cfaf1e29b46c03b.rlib,libonce_cell-0f6ca19213337e7c.rl
ib}" "/nix/store/464j7y43imzxc94955xgysxxcnadaah0-rustc-1.85.0/lib/rustlib/x86_64-unknown-linux-gnu/lib/{libstd-6569c13f8b87c66
b.rlib,libpanic_unwind-0c3f40eca452be75.rlib,libobject-23f98b95759ee8af.rlib,libmemchr-27a2577f03a7e63b.rlib,libaddr2line-c5094
2a0aee8d89d.rlib,libgimli-84fe1c3817dd2178.rlib,librustc_demangle-ace40700b6a7c973.rlib,libstd_detect-a515e52383d38ea8.rlib,lib
hashbrown-fa940357dffb30be.rlib,librustc_std_workspace_alloc-12055ddd6f34ba2d.rlib,libminiz_oxide-d13c10759e64ad67.rlib,libadle
r-ba627c69dd8676b6.rlib,libunwind-7116821b76ea6aa9.rlib,libcfg_if-3b94af249756a5d7.rlib,liblibc-9c3ed48fb1de302c.rlib,liballoc-
623d2d8b4f2c37ca.rlib,librustc_std_workspace_core-0377a0b832bfc874.rlib,libcore-b08c35bac43a364c.rlib,libcompiler_builtins-b15b
4ec8580e3dd7.rlib}" "-Wl,-Bdynamic" "-lpipewire-0.3" "-lpipewire-0.3" "-ldisplay-info" "-lseat" "-ludev" "-linput" "-ldl" "-lxk
bcommon" "-ldl" "-lgbm" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc" "-Wl,--eh-frame-hdr" "-Wl,-z,noexecstack" "-L"
 "/nix/store/48jgb2ihb9ccgqmwcdgwkg8vbrwcn169-systemd-minimal-libs-257.3/lib" "-L" "/nix/store/22svsj3ivbibhksnd7lwawdnk7wgbz9a
-seatd-0.9.1/lib" "-L" "/nix/store/sy0n827w0ccnl4izd50qbijmnc9np852-libdisplay-info-0.2.0/lib" "-L" "/build/17lz53zj5vqxi9335j1
h6dqqxvvvaqsc-source/target/x86_64-unknown-linux-gnu/release/build/libspa-16c9ef54cf129725/out" "-L" "/nix/store/nvj68rjl3f4900
dba9gxynabzqg5m85n-pipewire-1.4.0/lib" "-L" "/build/17lz53zj5vqxi9335j1h6dqqxvvvaqsc-source/target/x86_64-unknown-linux-gnu/rel
ease/build/libspa-sys-fd5fedf6a032c2d7/out" "-L" "/nix/store/nvj68rjl3f4900dba9gxynabzqg5m85n-pipewire-1.4.0/lib" "-L" "/nix/st
ore/464j7y43imzxc94955xgysxxcnadaah0-rustc-1.85.0/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-o" "/build/17lz53zj5vqxi9335j1h6d
qqxvvvaqsc-source/target/x86_64-unknown-linux-gnu/release/deps/fht_compositor-eb3ae42a36a22e12" "-Wl,--gc-sections" "-pie" "-Wl
,-z,relro,-z,now" "-Wl,-O1" "-nodefaultlibs" "-Wl,--push-state,--no-as-needed" "-lEGL" "-lwayland-client" "-Wl,--pop-state" "/b
uild/17lz53zj5vqxi9335j1h6dqqxvvvaqsc-source/target/x86_64-unknown-linux-gnu/release/deps/fht_compositor_audit_data.o" "-Wl,--u
ndefined=AUDITABLE_VERSION_INFO"
       >   = note: some arguments are omitted. use `--verbose` to show all linker arguments
       >   = note: /nix/store/dh0m67pv8snfcx1ic18jzp0x4cb10zhc-binutils-2.43.1/bin/ld: cannot find -lgbm: No such file or direc
tory
       >           collect2: error: ld returned 1 exit status
       >           
       >
       > warning: `fht-compositor` (bin "fht-compositor") generated 2 warnings
       > error: could not compile `fht-compositor` (bin "fht-compositor") due to 1 previous error; 2 warnings emitted
       For full logs, run 'nix log /nix/store/z1pmndvigc2kmz2jbjcgvms7paz4dh3q-fht-compositor-8f0c359.drv'.
```

--- a/flake.nix
+++ b/flake.nix
@@ -32,6 +32,7 @@
       seatd,
       libxkbcommon,
       mesa,
+      libgbm,
       pipewire,
       dbus,
       wayland,
@@ -72,7 +73,7 @@
 
         nativeBuildInputs = [rustPlatform.bindgenHook pkg-config installShellFiles];
         buildInputs =
-          [libGL libdisplay-info libinput seatd libxkbcommon mesa wayland]
+          [libGL libdisplay-info libinput seatd libxkbcommon mesa libgbm wayland]
           ++ lib.optional withXdgScreenCast dbus
           ++ lib.optional withXdgScreenCast pipewire;
 

